app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def index
+    greetings = Greeting.all
+    render json: greetings
+  end
```

Comment:
- [ ] You are fetching all greetings and displaying all greetings also but that is not the expected behavior. You are expected to fetch **ONE** random message from the API and display only that message on the frontend. Kindly fix that.

  You can use this expression to fetch random messages: `Greeting.all.sample`
  
  ![all-greet](https://user-images.githubusercontent.com/36542986/189881804-be39334e-2e9c-4fbc-a1d7-ad38ba2d8a6c.jpg)